src/components/TopBar/Crumb.js

```diff
@@ -0,0 +1,150 @@
+// @flow
+
+import React, { PureComponent } from 'react'
+import styled from 'styled-components'
+import type { Account, TokenAccount } from '@ledgerhq/live-common/lib/types/account'
+import { translate } from 'react-i18next'
+import IconAngleDown from 'icons/AngleDown'
+import IconCheck from 'icons/Check'
+
+import CryptoCurrencyIcon from '../CryptoCurrencyIcon'
+import DropDown from '../base/DropDown'
+import Button from '../base/Button'
+import type { DropDownItemType } from '../base/DropDown'
+
+type Props = {
+  items: Array<DropDownItemType>,
+  account: Account,
+  token?: TokenAccount,
+  push: string => void,
+}
+
+const Item = styled.div`
+  font-family: 'Open Sans';
+  font-size: 13px;
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  padding: 12px;
+  min-width: 200px;
+  color: ${p => (p.isActive ? p.theme.colors.dark : p.theme.colors.smoke)};
+  > :first-child {
+    margin-right: 10px;
+  }
+
+  &:hover {
+    background: ${p => p.theme.colors.lightGrey};
+    border-radius: 4px;
+  }
+`
+
+const TextLink = styled.div`
+  font-family: 'Open Sans';
+  font-size: 12px;
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+
+  > :first-child {
+    margin-right: 8px;
+  }
+
+  > :nth-child(2) {
+    padding: 0px;
+    &:hover {
+      background: transparent;
+      text-decoration: underline;
+    }
+    margin-right: 7px;
+  }
+`
+const AngleDown = styled.div`
+  width: 14px;
+  height: 14px;
+  border-radius: 20px;
+  text-align: center;
+  line-height: 14px;
+
+  &:hover {
+    background: ${p => p.theme.colors.fog};
+  }
+`
+
+const Check = styled.div`
+  color: ${p => p.theme.colors.wallet};
+  flex: 1;
+  text-align: right;
+  margin-left: 20px;
+`
+
+class Crumb extends PureComponent<Props> {
```

Comment:
in the current implementation, since things are `.map(` ed by the parent, there are no optim of this Pure. it will always re-render